src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,148 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> last;
+    private Node<T> first;
+
     @Override
     public void add(T value) {
+        addNew(value);
```

Comment:
Maybe we don't need the additional method "addNew(T value)" if in method "add(T value)" we call only it. We can move logic from  "addNew(T value)" to "add(T value)".

Reply:
Yea, you right, thanks.)